cypress/e2e/adapters.ts

```diff
@@ -32,11 +33,24 @@ describe('Adapter install', () => {
         "mapping": "{repoUrl:`https://kendraio-adapter.kendraio.now.sh/`, name:`bloomen`}"
       },
       {
-        "type": "dispatch",
-        "action": "installAdapter"
+        "type": "actions",
+        "buttons": [
+          {
+            "label": "Import Adapter",
+            "color": "default",
+            "blocks": [
+
+              {
+                "type": "dispatch",
+                "action": "installAdapter"
+              }]
+          }]
       }
     ]);
-    cy.wait(1000); // give the adapter time to install.
+    
+    let currentConfig=localStorage.getItem('core.variables.settings');
+    cy.log("Local:").log(currentConfig);
+    cy.contains('Import Adapter').click().wait(['@bloomen']).wait(2000); // make sure that the data is downloaded and give the adapter time to install.
```

Comment:
If the settings are not in place, this call never happens, because it is blocked in the code.